app/views/requests.erb

```diff
@@ -0,0 +1,17 @@
+<h1>Requests</h1>
+<ul>
```

Comment:
If you don't want to differentiate in the controller would you want to differentiate here? I.e. separate lists or something like that?